src/index.html

```diff
@@ -76,8 +76,164 @@ <h1 class="c-banner__title">You have a concept,<br><span class="h-blue">we'll ma
     </div>
 
     <div class="container-fluid">
-        <div class="container">
+        <div class="container l-container-padding">
+
+            <div class="c-section row" id="what-we-do">
+                <div class="l-no-padding col-sm-10 col-md-8 col-lg-6">
+                    <h2 class="c-section__title">What we do</h2>
+                    <div class="c-section__divider"></div>
+                    <p class="c-section__description">We will help you take your business to the next level. Our team consists of people that joined profession with their passion.</p>
+                </div>
+            </div>
+        
+            <div class="c-slide-nav">
+                <ul class="c-slide-nav__list row">
+                    <li class="c-slide-nav__item col-3">
+                        <a class="c-slide-nav__link c-slide-nav__link--active js-service" data-slide="0" role="button">
+                            <div class="c-slide-nav__icon">
+                                <img class="c-slide-nav__icon-img" src="img/apps-gray.jpg" alt="Apps">
+                            </div>
+                            <h1 class="c-slide-nav__title">App Development</h1>
```

Comment:
IMHO this should not be `h1`. If we want to use `h*` maybe `h4`?

- [ ] Change `h1`

Reply:
It is done for SEO purposes. H1 is analysed wrt. key phrases.